softdevice_controller/include/sdc_hci_vs.h

```diff
@@ -1259,12 +1316,12 @@ uint8_t sdc_hci_cmd_vs_read_average_rssi(const sdc_hci_cmd_vs_read_average_rssi_
 
 /** @brief Set Central ACL event spacing.
  *
- * On the central, sets the time ACL connections are spaced apart given that they are using the same
- * connection interval.
+ * On the central, sets the time ACL connections are spaced apart given that they are
```

Comment:
"sets the time ACL connections are spaced apart, assuming they are using the same connection interval" ?

(guessing based on the code)


"given that" means "since" https://en.wiktionary.org/wiki/given_that

Reply:
Feel free to update the documentation :)